examples/flutter_gallery/lib/gallery/drawer.dart

```diff
@@ -49,78 +62,102 @@ class GalleryDrawer extends StatelessWidget {
       child: new Block(
         children: <Widget>[
           new DrawerHeader(
-            child: new Center(
-              child: new Padding(
-                padding: const EdgeInsets.all(16.0),
-                child: new Image.asset(
-                 'packages/flutter_gallery_assets/drawer_logo.png',
-                  fit: ImageFit.contain
-                )
-              )
+            decoration: new FlutterLogoDecoration(
+              margin: new EdgeInsets.fromLTRB(12.0, 12.0 + ui.window.padding.top, 12.0, 12.0),
```

Comment:
The reason I didn't do this is that I don't want to add left or right padding. The contract here is that this widget will overlap the top status bar but not anything else. So if there was left padding, adding the left padding would mean it was double displaced.